src/EPPlusTest/Core/Range/RangeTest.cs

```diff
@@ -315,6 +315,43 @@ public void ValidateDeleted()
                 ws.Tables.Add(ws.Cells["D4:E5"], "Table1");
             }
         }
+
+        [TestMethod]
+        public void RangeIsEmptyTest()
```

Comment:
Add test for threaded comments. Check that Value is null for cells with a comment or a threaded comment, so its hit by the CellStoreEnumerator.